packages/theme/pages/Category.vue

```diff
@@ -259,11 +270,18 @@ export default {
       }
     };
 
+    onMounted(async () => {
+      try {
+        menu.value = await $spree.api.getMenus({menuType: 'header', menuName: 'Main menu', locale: locale.value});
```

Comment:
Could we maybe create a custom composable, e.g. `useMenus()` to load and cache menus between components?